test/parallel/test-runner-cli.js

```diff
@@ -428,3 +428,31 @@ for (const isolation of ['none', 'process']) {
   assert.strictEqual(child.status, 0);
   assert.strictEqual(child.signal, null);
 }
+
+{
```

Comment:
Please move these new tests to a new file - `test/parallel/test-runner-worker-id.js`.